shared/fs/banner/out-of-space-banner/container.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from 'react'
+import Banner, {height} from './index'
+import * as FsGen from '../../../actions/fs-gen'
+import * as RowTypes from '../../row/types'
+import {namedConnect} from '../../../util/container'
+import {isMobile} from '../../../constants/platform'
+
+type OwnProps = {||}
+
+const mapStateToProps = state => ({})
+
+const mapDispatchToProps = dispatch => ({
+  // TODO: set this to a "retry sync" action
+  onRetry: () => dispatch(FsGen.createDriverEnable({})),
+})
+
+const ConnectedBanner = namedConnect<OwnProps, _, _, _, _>(
+  mapStateToProps,
+  mapDispatchToProps,
+  (s, d, o) => ({...o, ...s, ...d}),
+  'OutOfSpaceBanner'
+)(Banner)
+
+export default ConnectedBanner
+
+// TODO do we actually need this?
```

Comment:
Not sure; My understanding is this banner will stay there even if user scrolls down in the folder, which means this won't be part of Rows. If that's correct, then we don't need this (and we can remove the `List2` comment at the beginning of the component file).

@cecileboucheron could you confirm that the banner [here](https://zpl.io/2EPL8on) doesn't scroll with the folder content?

Reply:
So as Cécile mentioned in chat, this should not be part of the row on desktop, but part of the row on mobile. i.e. we still need these `asRows` stuff as well as keeping height constant.

Reply:
And this is why I shouldn't force push so often 🙈

I've put back said code.